changes/issue-10126-mdm-info

```diff
@@ -1 +0,0 @@
-- Added MDM solution name to `host.mdm`in API responses
```

Comment:
Should this have been deleted here?

Reply:
Yes, this is on `main`. I will include in 4.29.0 release notes.